hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -178,7 +179,7 @@ private void scheduleRelogin(Throwable error) {
     RELOGIN_EXECUTOR.schedule(() -> {
       try {
         provider.relogin();
-      } catch (IOException e) {
+      } catch (Throwable e) {
```

Comment:
Yeah, I think it makes sense. IIRC, netty swallows the throwable silently, so it's difficult to figure out what's wrong.

Reply:
I think other parts catching only IOException is just fine, the uncatched exception will throw upward